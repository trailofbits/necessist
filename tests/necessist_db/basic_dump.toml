args = ["--root", "examples/basic", "--dump"]
stderr = ""

[bin]
name = "necessist"

[status]
code = 0

[env.add]
TRYCMD="1"
