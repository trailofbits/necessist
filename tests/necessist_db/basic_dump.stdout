examples/basic/src/lib.rs:4:5-4:12: `n += 1;` passed
Warning: More output would be produced with --verbose
